### PR TITLE
Populate readme.md

### PR DESCRIPTION
**Pull request by tehp**
Kept most setup information, revamped everything else.

### CODE REVIEW

**Comment by meyer9** [comment]
This should have the new installation instructions for all platforms:

```
virtualenv env
source env/bin/activate
pip install -r requirements.txt
```

**Comment by MaxLeiter** [comment]
All links need to be changed to this repo

**Comment by meyer9** [comment]
Feature section needs rewording to be proper English.

**Comment by meyer9** [comment]
:-1: see my first comment

[Rejected with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jtdroste** [comment]
I think this can be squashed into one commit.  Can you please rebase -i HEAD~3?

**Comment by jtdroste** [comment]
Testing github notifier...

**Comment by meyer9** [comment]
Squashed the commits

File: README.md [hunk 1]
```diff
@@ -1 +1,203 @@
...
+
+## Installation
+
+### Requirements
+
+- [Python 2.7.x](http://docs.python-guide.org/en/latest/starting/installation/)
+- [pip](https://pip.pypa.io/en/stable/installing/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) (Optional)
+- [docker](https://docs.docker.com/engine/installation/) (Optional)
```
Comment (jtdroste) [inline README.md]: Should be removed

**Comment by jtdroste** [comment]
Other than that one line, lgtm

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by meyer9** [comment]
:-1:

[Rejected with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jmeyer2k** [comment]
```
  - wget https://protobuf.googlecode.com/files/protobuf-2.4.1.tar.gz
  - tar -xzvf protobuf-2.4.1.tar.gz
  - pushd protobuf-2.4.1 && ./configure --prefix=/usr && make && sudo make install && popd
```

**Comment by wchill** [comment]
:-1: still needs Docker line removed.

[Rejected with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by jtdroste** [comment]
Edit: Spoke too soon.  Sqush pls?

**Comment by wchill** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**